.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: '🐛 Bug report'
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue :pray:.
+
+        This issue tracker is for reporting bugs found in `remix-cookbook.com` (https://github.com/remix-cookbook/remix-cookbook.com).
+        If you have a question about how to achieve something and are struggling, please post a question
+        inside of `remix-cookbook.com` Discussions tab: https://github.com/remix-cookbook/remix-cookbook.com/discussions
+
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - `remix-cookbook.com` Issues tab: https://github.com/remix-cookbook/remix-cookbook.com/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
+         - `remix-cookbook.com` closed issues tab: https://github.com/remix-cookbook/remix-cookbook.com/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed
+         - `remix-cookbook.com` Discussions tab: https://github.com/remix-cookbook/remix-cookbook.com/discussions
+
+        The more information you fill in, the better the community can help you.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of the challenge you are running into.
+    validations:
+      required: true
+  - type: input
+    id: link
+    attributes:
+      label: Your Example Website or App
+      description: |
+        Which website or app were you using when the bug happened?
+        Note:
+        - Your bug will may get fixed much faster if we can run your code and it doesn't have dependencies other than the `remix-cookbook.com` npm package.
```

Comment:
This repo is not an NPM package. We should rephrase this line.

Reply:
Updated